plotting/map.py

```diff
@@ -390,6 +401,7 @@ def load_data(self):
                 contour_unit = vc.unit
                 contour_name = vc.name
                 contour_data.append(d)
+                self.contour_data = contour_data
```

Comment:
We set `self.contour_data = contour_data` here and outside of the _if_ block on line 408. We probably don't need both so I think we should just keep the one on 408 so that `self.contour_data` will be empty when there's no contours.